costco-deal-hunter.py

```diff
@@ -0,0 +1,100 @@
+# selenium allows us to interact with HTML elements, such as clicking on a page using automation; in this case we will use it for specifying location on the Costco website
+from selenium import webdriver
+# to let us type stuff from our code we need to import Keys:
+from selenium.webdriver.common.keys import Keys
+# selenium module that lets us make the web driver wait until a certain condition is met (browser is loaded enough)
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import NoSuchElementException
+# pandas to be used for data storage and analysis
+import pandas as pd
+
+# must specify where the web driver is located and create a variable that will call on the web driver for chrome
+path = "/Applications/chromedriver-92" 
+driver = webdriver.Chrome(path)
+# to make code easier to read, create variable for the url you want to use
+url = "https://www.costco.ca/coupons.html"
+
+# initializing lists
+names = []
+prices = []
+savings = []
+start_dates = []
+end_dates = []
```

Comment:
Instead of initializing 5 lists and appending each column of data into them, you can initialize one list of `data`, represent the data as one dictionary (where your keys are name/savings/price/start date/end date), and then append the data to your single initialized list. The end result is the same but since each iteration is a product and its metadata, it's better to represent the product as one set of key/value pairs